### PR TITLE
[Components V3] Action icons

### PR DESCRIPTION
**Pull request by jeremie-lucca**
## Description

Action icon UI update

-----

We will probably update button component in the future to cover icons without text, which will make Action Icons deprecated. For now, we can allow to have Ghost mode (Text) by default and Outlined as option to avoid breakings.

image

-----

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2098/storybook

File: packages/scss/src/components/actionIcon/mods.scss [hunk 1]
```diff
@@ -17,39 +19,20 @@
 @mixin outlined {
 	box-shadow: 0 0 0 1px var(--palettes-grey-400);
```
Comment (MarcFairbrother) [inline packages/scss/src/components/actionIcon/mods.scss:20]:
```suggestion
	box-shadow: 0 0 0 var(--commons-divider-width) var(--palettes-grey-400);
```

File: packages/scss/src/components/actionIcon/mods.scss [hunk 2]
```diff
@@ -58,16 +41,32 @@
 		box-shadow: 0 0 0 1px var(--palettes-error-400), var(--commons-boxShadow-XS);
```
Comment (MarcFairbrother) [inline packages/scss/src/components/actionIcon/mods.scss:41]: Idem

File: packages/scss/src/components/actionIcon/mods.scss [hunk 3]
```diff
@@ -58,16 +41,32 @@
...
 	}
 
-	&:focus {
-		box-shadow: 0 0 0 1px var(--palettes-error-400), 0 0 0 4px var(--palettes-error-100);
+	&:focus-visible {
+		background-color: var(--palettes-error-50);
+		box-shadow: 0 0 0 1px var(--palettes-error-400), var(--commons-boxShadow-XS);
```
Comment (MarcFairbrother) [inline packages/scss/src/components/actionIcon/mods.scss:46]: Idem

**Review by MarcFairbrother — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2098/storybook

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-2098/storybook

**Review by MarcFairbrother — APPROVED** [review]

**Pull request merged**